src/NServiceBus.AcceptanceTests/Timeouts/When_endpoint_does_not_disable_dtc.cs

```diff
@@ -0,0 +1,113 @@
+﻿
+namespace NServiceBus.AcceptanceTests.Timeouts
+{
+    using System;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTesting.Support;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Transports;
+    using NUnit.Framework;
+
+    class When_endpoint_does_not_disable_dtc : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public void Endpoint_should_start_with_msmq()
+        {
+            var context = new Context();
+
+            Scenario.Define(context)
+                .WithEndpoint<MsmqEndpoint>()
+                .Done(c => c.EndpointsStarted)
+                .Run();
+
+            Assert.IsTrue(context.EndpointsStarted);
+        }
+
+        [Test]
+        public void Endpoint_should_start_with_sqlTransport()
+        {
+            var context = new Context();
+
+            Scenario.Define(context)
+                .WithEndpoint<MsmqEndpoint>()
+                .Done(c => c.EndpointsStarted)
+                .Run();
+
+            Assert.IsTrue(context.EndpointsStarted);
+        }
+
+        [Test]
+        public void Endpoint_should_start_with_rabbitTransport()
+        {
+            var context = new Context();
+
+            var exception = Assert.Throws<AggregateException>(() => Scenario.Define(context)
```

Comment:
the test naming and placement doesn't really match with the actual test implementation